proto/api/v1/delegated_registry_config_service.proto

```diff
@@ -0,0 +1,80 @@
+syntax = "proto3";
+
+option go_package = "v1";
+
+option java_package = "io.stackrox.proto.api.v1";
+
+import weak "google/api/annotations.proto";
+import "api/v1/empty.proto";
+
+package v1;
+
+message DelegatedRegistryClustersResponse {
+    repeated DelegatedRegistryCluster clusters = 1;
+}
+
+message DelegatedRegistryCluster {
+    string id      = 1;
+    string name    = 2;
+    bool   isValid = 3;
+}
+
+// DelegatedRegistryConfig determines if and where scan requests are delegated to, such as kept in 
+// central services or sent to particular secured clusters
+message DelegatedRegistryConfig {
+    enum EnabledFor {
+        // delegate no scans
+        NONE     = 0;
+        // delegate all scans
+        ALL      = 1;
+        // delegate scans that match specific []registries
+        SPECIFIC = 2;
+    }
+
+    message DelegatedRegistry {
+        // registry + optional path, ie: quay.example.com/prod
+        string registry_path = 1;
+        // id of the cluster to delegate requests to
+        string cluster_id    = 2;
+    }
+
+    // determines if delegation is enabled for no registries, all registries, or specific registries
+    EnabledFor enabled_for       = 1;
+
+    // the default cluster to delegate ad-hoc requests to if no match found in []registries, not used
+    // if enabledFor is NONE
+    string default_cluster_id    = 2;
+
+    // list of registries that has different outcome depending on value of enabled_for, allows
```

Comment:
`list of registries that has different outcome` not sure I follow this

Reply:
I'll see if I can wordsmith that better - the registries in the list is interpreted differently depending on use case (ie: ad-hoc vs. informer event) and/or the value of `enabled for`